core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala

```diff
@@ -125,9 +125,13 @@ trait EntityApiService extends HttpService with PerRequestCreator with UserInfoD
     } ~
     path("workspaces" / "entities" / "copy" ) {
       post {
-        entity(as[EntityCopyDefinition]) { copyDefinition =>
-          requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
-            WorkspaceService.CopyEntities(copyDefinition, requestContext.request.uri))
+        parameters('linkExistingEntities.?) { (linkExistingEntities) =>
+          val linkExistingEntitiesBool = Try(linkExistingEntities.getOrElse("false").toBoolean).getOrElse(false)
```

Comment:
I want a prettier way to express this but so far my mental attempts are no better